master/master.cfg

```diff
@@ -73,6 +73,35 @@ def do_step_build_spl(step):
 def do_step_build_zfs(step):
     return do_step_build(step, 'buildzfs')
 
+style_factory = util.BuildFactory()
+
+style_factory.addStep(Git(repourl=zfs_repo, workdir="build/zfs",
+    mode="full", method="clobber", codebase="zfs",
+    logEnviron=False, getDescription=True,
+    description=["cloning"], descriptionDone=["cloned"]))
+style_factory.addStep(ShellCommand(command=["./autogen.sh"],
+    env={'PATH' : bin_path},
+    workdir="build/zfs", logEnviron=False,
+    decodeRC={0 : SUCCESS, 1 : FAILURE, 2 : WARNINGS, 3 : SKIPPED },
+    description=["autogen"], descriptionDone=["autogen"]))
+style_factory.addStep(ShellCommand(command=["./configure", "--with-config=srpm"],
+    env={'PATH' : bin_path},
+    workdir="build/zfs", logEnviron=False,
+    decodeRC={0 : SUCCESS, 1 : FAILURE, 2 : WARNINGS, 3 : SKIPPED },
+    description=["configuring"], descriptionDone=["configure"]))
```

Comment:
Went ahead and revised the build steps to use `bb-build-zfs.sh`. I will update the README shortly.